frontend/lib/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -91,7 +195,7 @@ export const DeckGlJsonChart: FC<PropsWithHeight> = props => {
               ? deck.mapStyle
               : deck.mapStyle[0])
           }
-          mapboxApiAccessToken={props.element.mapboxToken || props.mapboxToken}
+          mapboxApiAccessToken={element.mapboxToken || mapboxToken}
```

Comment:
nit: how about destructuring it already with descriptive names [in the lines above](https://github.com/streamlit/streamlit/pull/9377/files#diff-75e7f4652df3884dab3b3edaa2507d4393363d902403af9ac95889bd0ce88bc4R58-R62) where you already destructure the `props` and `element` fields:

```typescript
{ mapboxToken: propsMapboxToken } = props;
{ mapboxToken: elementMapboxToken } = element;
```

and then here: `mapboxApiAccessToken={elementMapboxToken || propsMapboxToken}`